src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,138 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    class Node<T> {
```

Comment:
```suggestion
    private class Node<T> {
```
and move it to the end of the class